pip-requirements.el

```diff
@@ -62,6 +62,10 @@
 (add-to-list 'auto-mode-alist
              `(,(rx "requirements" (zero-or-more anything) ".txt" string-end) . pip-requirements-mode))
 
+;;;###autoload
+(add-to-list 'auto-mode-alist
+`(,(rx "requirements" (zero-or-more anything) ".in" string-end) . pip-requirements-mode))
```

Comment:
Would this make sense as just `(,(rx "requirements.in") . pip-requirements-mode)`, or are you expecting `requirements-foobar.in` too?